sunpy/timeseries/tests/test_timeseriesbase.py

```diff
@@ -419,7 +421,7 @@ def test_remove_column(eve_test_ts):
     assert len(removed.columns) == removed.to_dataframe().shape[1]
 
     # Check that removing a non-existent column errors
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match=re.escape("Given column name (random column name) not in list of columns ['XRS-B proxy', 'XRS-A proxy', 'SEM proxy', '0.1-7ESPquad', '17.1ESP', '25.7ESP', '30.4ESP', '36.6ESP', 'darkESP', '121.6MEGS-P', 'darkMEGS-P', 'q0ESP', 'q1ESP', 'q2ESP', 'q3ESP', 'CMLat', 'CMLon', 'x_cool proxy', 'oldXRSB proxy']")):
```

Comment:
There is no need to list every column, just catch the start of the error message.